### YamlMime:ManagedReference
items:
- uid: System.Threading.ThreadStartException
  id: ThreadStartException
  children: []
  langs:
  - csharp
  name: ThreadStartException
  nameWithType: ThreadStartException
  fullName: System.Threading.ThreadStartException
  type: Class
  summary: "Исключение, возникающее, когда происходит сбой в управляемом потоке после запуска базового потока операционной системы, но до готовности к выполнению кода пользователя потока."
  remarks: "Исключение, возвращенных <xref:System.Exception.InnerException%2A>свойстве содержатся сведения о причине сбоя.</xref:System.Exception.InnerException%2A>"
  syntax:
    content: 'public sealed class ThreadStartException : SystemException'
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
